mpsl/CHANGELOG.rst

```diff
@@ -12,6 +12,10 @@ Main branch
 
 All the notable changes included in the main branch are documented in this section.
 
+Bug fixes
+=========
+* Fixed an issue where the scheduler could put events in the past. (DRGN-17923, DRGN-18105)
```

Comment:
DRGN-17923 is "Peripheral side: CIS Creation procedure: implement state sequence in LL"

Reply:
Maybe that is intended :)